app/src/main/java/de/westnordost/streetcomplete/osm/surface/Surface.kt

```diff
@@ -0,0 +1,80 @@
+package de.westnordost.streetcomplete.osm.surface
+
+import de.westnordost.streetcomplete.osm.surface.Surface.*
+
+data class SurfaceAndNote(val surface: Surface?, val note: String? = null)
+
+enum class Surface(val osmValue: String?) {
+    ASPHALT("asphalt"),
+    CONCRETE("concrete"),
+    CONCRETE_PLATES("concrete:plates"),
+    CONCRETE_LANES("concrete:lanes"),
+    FINE_GRAVEL("fine_gravel"),
+    PAVING_STONES("paving_stones"),
+    COMPACTED("compacted"),
+    DIRT("dirt"),
+    MUD("mud"),
+    SETT("sett"),
+    UNHEWN_COBBLESTONE("unhewn_cobblestone"),
+    GRASS_PAVER("grass_paver"),
+    WOOD("wood"),
+    WOODCHIPS("woodchips"),
+    METAL("metal"),
+    GRAVEL("gravel"),
+    PEBBLES("pebblestone"),
+    GRASS("grass"),
+    SAND("sand"),
+    ROCK("rock"),
+    CLAY("clay"),
+    ARTIFICIAL_TURF("artificial_turf"),
+    TARTAN("tartan"),
+    // generic surfaces
+    PAVED("paved"),
+    UNPAVED("unpaved"),
+    GROUND("ground"),
+
+    // extra values, handled as synonyms (not selectable)
+    EARTH("earth"), // synonym of "dirt"
+    CHIPSEAL("chipseal"), // subtype/synonym of asphalt
+    METAL_GRID("metal_grid"), // not exactly the same info as metal
+    // this way than without info
```

Comment:
These comments look like they should form one sentence. Either concatenate them, or fix the second one's indentation, or remove the second one (as it's not needed IMO):

```suggestion
    METAL_GRID("metal_grid"), // not exactly the same info as metal
```